### ISSUE TITLE
[Feature] Header Design Discrepancy with System Guidelines

### ISSUE THREAD

**Issue by Silvia-Wachira** [creativecommons/reversionary-rights#31]
## Problem
The header design is inconsistent with the design system hence leading to a  less cohesive user experience across pages.

![screenshot](https://github.com/user-attachments/assets/cf5e2001-52b5-45a0-9ac0-e1576c975fb4)
![screenshot2](https://github.com/user-attachments/assets/06bb6675-2adb-4eb9-bbbf-077a06c8fc73)

## Description
This feature addresses inconsistencies between the header and the design system. By implementing design system guidelines , the update ensures better alignment and a consistent interface for users.

## Additional context
This update is part of a broader effort to standardize all UI components according to the design system, ensuring that the application remains visually consistent as new features are developed. This also makes it easier for future contributors to maintain consistency.
## Implementation

- [x] I would be interested in implementing this feature.

**Comment by possumbilities** [comment]
This seems to be a valid issue, but would require the full implementation of Vocabulary to accomplish. Moving to `status: blocked` until that is done.

**Comment by Silvia-Wachira** [comment]
Okay.That's noted

**Comment by possumbilities** [comment]
@Silvia-Wachira You could make an Issue regarding "implement Vocabulary design system" detailing that requirement?

**Comment by Silvia-Wachira** [comment]
So it will not only be about the header but the whole site, right?

**Comment by possumbilities** [comment]
@Silvia-Wachira Yeah, the entire site would need Vocabulary to then get all the things that come with Vocabulary (even if iteratively).
